### ISSUE TITLE
pytest fixture accounts does not work

### ISSUE THREAD

**Issue by 4gn3s** [pipermerriam/populus#97]
* Populus Version: 0.8.0
* OS: linux
* py-ethereum Version: 1.3.6

### What was wrong?

When trying to use `accounts`, I get the following error:
```
>   return tuple("0x" + tester.encode_hex(account) for account in tester.accounts)
E   AttributeError: 'module' object has no attribute 'encode_hex'
```
This is caused by interface changes in pyethereum. `encode_hex` should be imported from `ethereum.utils`.

I would gladly provide a PR to fix this.

### Cute animal picture
![cute animal](https://media.giphy.com/media/w6c5NOfPPhtv2/giphy-facebook_s.jpg)

**Comment by pipermerriam** [comment]
> I would gladly provide a PR to fix this.

That would be great.

**Comment by pipermerriam** [comment]
Will be fixed when #105 is merged.
